proto/osmosis/gamm/v1beta1/query.proto

```diff
@@ -146,9 +146,8 @@ message QuerySpotPriceResponse {
 
 //=============================== EstimateSwapExactAmountIn
 message QuerySwapExactAmountInRequest {
-  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
-  uint64 pool_id = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
-  string token_in = 3 [ (gogoproto.moretags) = "yaml:\"token_in\"" ];
+  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  string token_in = 2 [ (gogoproto.moretags) = "yaml:\"token_in\"" ];
```

Comment:
Thanks for catching this, pretty funny mistake lol!

However, please don't change the field numbers in proto files like this, lets keep them at the old field numbers, this makes it so legacy clients don't have to update, to 'just work' with the changed proto.

Reply:
Yep, I'll return old field number.